src/main/webapp/map.js

```diff
@@ -2,15 +2,16 @@ var map;
 var infoWindow;
 var shopInfo;
 var userPos;
-// require('dotenv').config();
+var geocoder;
+var markers = [];
 
 /** Creates a map that shows all coffee shops around the user. */
 function createMap() {
   // Set default location at new york city
   var newyork = new google.maps.LatLng(40.7128, -74.0060);
```

Comment:
Suggestion: make this a variable e.g. "default_location", so that if you need to change it, only one update is required.